web/static/auth0/connect/index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>AutoKitteh - Auth0 - New Connection</title>
+    <link
+      rel="apple-touch-icon"
+      sizes="180x180"
+      href="/static/apple-touch-icon.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="32x32"
+      href="/static/favicon-32x32.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="16x16"
+      href="/static/favicon-16x16.png"
+    />
+    <link rel="manifest" href="/static/site.webmanifest" />
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+    <meta name="robots" content="noindex, nofollow" />
+  </head>
+
+  <body>
+    <a href="/" target="_self">
+      <img class="banner" src="/static/banner.svg" alt="AutoKitteh" />
+    </a>
+
+    <table class="title">
+      <td><img class="logo" src="/static/images/auth0.svg" alt="Logo" /></td>
+      <td>
+        <h1>Auth0 - Create a New Connection</h1>
+      </td>
+    </table>
+
+    <p class="info">Information:</p>
+    <ul class="links">
+      <li>
+        <a
+          href="https://docs.autokitteh.com/integrations/auth0/config"
+          target="_blank"
+          class="info"
+        >
+          AutoKitteh guide: configuring the Auth0 integration
+        </a>
+      </li>
+      <li>
+        <a
+          href="https://auth0.com/docs/authenticate/identity-providers/enterprise-identity-providers/oauth2"
```

Comment:
404 page not found - We couldn't find the page you were looking for.

Reply:
Removed for now, will add in a different PR